### PR TITLE
replace the new layout section description 

### PR DESCRIPTION
**Pull request by Luis-RG**
replace the new layout section description text with a more accurate one and changes the translation key to replace the current text in all languages.

### CODE REVIEW

**Comment by Luis-RG** [comment]
I am not sure about the .html_safe part though...

File: config/locales/en.yml [hunk 1]
```diff
@@ -246,6 +246,7 @@ en:
         new_layout: "New layout"
         description_roadmap: "Sharetribe is currently in the process of rolling out a new and improved layout. During the transition period, you can select which parts of your marketplace should continue using the old layout using the menu below. New features will only be available on the new layout."
         description_process: "Support for the old layout will be removed in the future. The Sharetribe team will notify you before this happens. Questions? Contact the team via %{support_link}."
```
Comment (rap1ds) [inline config/locales/en.yml]: Now that `description_process` and `description_roadmap` aren't used anymore, they can and should be removed from the `en.yml` file.

File: app/views/admin/communities/new_layout.haml [hunk 2]
```diff
@@ -17,8 +17,7 @@
   .row
     .col-12
       %h2= t("admin.communities.new_layout.new_layout")
-      %p= t("admin.communities.new_layout.description_roadmap")
-      %p= t("admin.communities.new_layout.description_process", support_link: support_link).html_safe
```
Comment (rap1ds) [inline app/views/admin/communities/new_layout.haml]: `support_link` is not used anymore, so it should be removed also from the line 7.

**Comment by rap1ds** [comment]
@Luis-RG No need to use `.html_safe` in this case because the `support_link` inside the translation is removed. So it's good now as it is 👍

**Pull request merged**